### PR TITLE
Clarified Timer::getTime() return value

### PR DESCRIPTION
**Pull request by neilmenon**
`Timer::getTime()` returns seconds, with milliseconds included in the precision.
https://github.com/uva-cs/pdr/blob/8812f9ac96e8b0ba125b151ddc8eb21f83542fbf/labs/lab06/code/timer.cpp#L27-L32

### CODE REVIEW

File: labs/lab06/index.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ Shell scripts are useful when one needs to call a large number of Unix commands
 Write a shell script `averagetime.sh` that will prompt the user for the dictionary and grid file names used by your word puzzle executable, which must be called `a.out`.  It will then run the program five times using those parameters.  It will record the time of each execution run, and, once the runs are completed, print out the average run time.  Note that you have not yet seen conditions (`if` or `case`) or loops (`for` or `while`) in shell scripts, so we do not expect your script to have either of these -- you should just have 5 separate commands without a loop.
 
 Modify your word puzzle program to print out the total time taken as the last line of output.
-Floating point arithmetic in bash is quite complicated, so try to use integers when possible -- you may want to use the `Timer::getTime()` method, which returns milliseconds.
+Floating point arithmetic in bash is quite complicated, so try to use integers when possible -- you may want to use the `Timer::getTime()` method, which includes milliseconds in its return value.
```
Comment (50Wliu) [inline labs/lab06/index.md]: Let me quickly confirm that this is the case

Comment (50Wliu) [reply labs/lab06/index.md]: How about we rephrase this to "you may want to multiply the result from `Timer::getTime()` by 1000 to obtain the number of milliseconds taken"

Comment (neilmenon) [reply labs/lab06/index.md]: Sounds good!

**Comment by aaronbloomfield** [comment]
Thanks for the PR, Neil!
